wabt/src/main/java/com/dylibso/chicory/wabt/Wat2Wasm.java

```diff
@@ -30,11 +30,9 @@
 public final class Wat2Wasm {
     private static final Logger logger = new SystemLogger();
     private static final com.dylibso.chicory.wasm.Module wasmModule =
-            Module.builder(Wat2Wasm.class.getResourceAsStream("/wat2wasm"))
-                    .withInitialize(false)
-                    .withStart(false)
-                    .build()
-                    .wasmModule();
+            com.dylibso.chicory.wasm.Module.builder(Wat2Wasm.class.getResourceAsStream("/wat2wasm"))
```

Comment:
I'm not 100% convinced of having 2 `Module` classes around, but I don't have a "clearly better" proposal to change the name.

Reply:
I think we can remove `runtime.Module` as it's basically just a builder for `Instance`.

Reply:
I'm trying this approach, I'll update this PR as soon as I'm done, thanks for the suggestion!

Reply:
Done, I'm happy with the result!